digital_asset_types/src/dapi/common/asset.rs

```diff
@@ -151,6 +151,35 @@ pub fn safe_select<'a>(
         .and_then(|v| v.pop())
 }
 
+#[cfg(feature = "fetch-raw-fields")]
+async fn process_raw_fields(
```

Comment:
How will this function be used?

Reply:
Right now, I am using it for testing purpose. Its disabled due the compile/build time flag. However, based on a new optional field in the api, we will extract the raw fields or the sanitised fields